core/tests/revert-test/tests/revert-and-restart-en.test.ts

```diff
@@ -188,51 +241,37 @@ class ExtNode {
     // If enableConsensus is set, the node will use consensus P2P network to fetch blocks.
     public static async spawn(logs: fs.WriteStream, enableConsensus: boolean): Promise<ExtNode> {
         let env = fetchEnv(extEnv);
-        console.log(`DATABASE_URL = ${env.DATABASE_URL}`);
         let args = [];
         if (enableConsensus) {
             args.push('--enable-consensus');
         }
-        let proc = spawn('./target/release/zksync_external_node', args, {
-            cwd: env.ZKSYNC_HOME,
+
+        // Run server in background.
+        runExternalNodeInBackground({
             stdio: [null, logs, logs],
-            env: {
-                ...env,
-                PATH: process.env.PATH
-            }
+            cwd: pathToHome,
+            env: env,
+            useZkInception: fileConfig.loadFromFile
         });
+
         // Wait until the node starts responding.
-        let tester: Tester = await Tester.init(
-            env.EN_ETH_CLIENT_URL,
-            `http://127.0.0.1:${env.EN_HTTP_PORT}`,
-            env.CONTRACTS_BASE_TOKEN_ADDR
-        );
+        let tester: Tester = await Tester.init(ethClientWeb3Url, enEthClientUrl, baseTokenAddress);
         while (true) {
             try {
                 await tester.syncWallet.provider.getBlockNumber();
                 break;
             } catch (err) {
-                if (proc.exitCode != null) {
-                    assert.fail(`node failed to start, exitCode = ${proc.exitCode}`);
-                }
+                // NOTE: not an infinite loop, as the test will eventually timeout.
```

Comment:
Seems, that fail fast is better. timeout usually is huge

Reply:
Now that the test uses a dedicated `background` utils, we can restore the previous check. Addressed in https://github.com/matter-labs/zksync-era/pull/2408/commits/459057591a02417a8b0f01f1199861ce041846f0